src/Dfe.Spi.UkrlpAdapter.Domain/Dfe.Spi.UkrlpAdapter.Domain.csproj

```diff
@@ -4,4 +4,8 @@
         <TargetFramework>netstandard2.0</TargetFramework>
     </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Dfe.Spi.Models" Version="1.0.6" />
```

Comment:
Might want the latest models lib in here... 1.0.8 I think?

Reply:
That is my next bit of work. Wanted to get the cache done independently first